tljh/installer.py

```diff
@@ -485,6 +486,10 @@ def main():
         nargs='*',
         help='Plugin pip-specs to install'
     )
+    argparser.add_argument(
+        '--temporary-page',
```

Comment:
If this is taking the `pid` of the process showing the temporary page, we should rename this to something like `--progress-page-server-pid` or similar? Could also add `type=int` to enforce it's an integer.